### DISCUSSION TITLE
[el-table] 拖动表头改变了列的宽度储存到本地

### DISCUSSION THREAD

**Discussion by xl495** [element-plus/element-plus#17809] [Ideas]
提供一个属性 拖拽后把目前的宽度储存到本地会不会好一些呢?  这样也算支持用户个性化

**Comment by HaceraI** [comment]
![image](https://github.com/user-attachments/assets/b21c74b2-e846-427a-a98b-6cd8788b90f4)

已经提供了相应的函数`header-dragend`，你可以监听并做自定义处理.
